docs/blog/posts/announcing-keptn-v2-release-candidate.md

```diff
@@ -0,0 +1,148 @@
+authors: [agardnerit,bacherfl]
+description: >
+  Announcing the Official Release Candidate for Keptn v2
+categories:
+  - Installation
+  - Upgrade
+comments: true
+---
+
+# Announcing the Official Release Candidate for Keptn v2
+
+The Keptn project is proud to announce a release candidate for what will become Keptn v2.
+This release candidate provides the project with enough end-user validation
+and bug-finding so that we can release Keptn v2.
+We anticipate (as long as no release blockers are identified) that Keptn
+v2 will be released approximately 1 month after this announcement.
+
+<!-- more -->
+
+## What Problem Does Keptn Solve?
+
+Deploying software using Kubernetes is, on the surface, easy.
+Just use `kubectl` or a GitOps solution like ArgoCD to deploy a YAML file and that's it, right?
+Well, no.
+Not normally.
+There is a lot more to ensuring a healthy deployment,
+and there are a lot of potential problems that can
+prevent the successful deployment of a Kubernetes application, such as:
+
+- Not enough resources (i.e. CPU, Memory) being available in the cluster.
+- An external dependency (e.g. a Database hosted outside the cluster) not being reachable.
+- Unhealthy state of your infrastructure (e.g. due to vulnerabilities).
+- The cluster or namespace not being monitored correctly - You would not want to deploy into the dark in this case.
+- Unexpected performance degradation after rolling out a new version of your application.
+
+This is the problem domain on which Keptn acts.
+By using Keptn, in combination with your standard deployment tooling or practices,
+you can move from "I guess it's OK" to "I know it's OK".
+Keptn allows you to wrap governance and automated checks around the deployment
+process to ensure that the end-to-end process of deploying is
+healthy and your application is meeting the SLOs you've defined.
+This is done by configuring Keptn to execute [KeptnTasks](../../docs/reference/crd-reference/task.md) and
+[KeptnEvaluations](../../docs/reference/crd-reference/evaluationdefinition.md) before and after the deployment of a workload.
+With these [KeptnTasks](../../docs/reference/crd-reference/task.md), you can implement checks that ensure your
+infrastructure is in a healthy state before a new version of a deployment is rolled out.
+This way, you can avoid the problems mentioned above by:
+
+- Verifying that your cluster has enough resources to run a workload.
+- Checking the reachability of external services that might be required by your workloads.
+- Ensuring that your cluster is monitored by your monitoring provider.
+- Verifying that your monitoring provider did not detect any open problems within your cluster.
+
+After a successful deployment of a workload, Keptn also provides the means to do post-deployment checks.
+These can be crucial in ensuring that the performance of your newly deployed workload meets your expectations,
+before promoting it into the next stage.
+
+## What’s New in Keptn v2?
```

Comment:
```suggestion
## What’s new in this build?
```
What do you think?  Some casual readers might see this as "how is Keptn v2 unlike Keptn v1?" -- it's kind of ambiguous.